snaps/qemu-op-tee-gadget/gadget.yaml

```diff
@@ -0,0 +1,31 @@
+volumes:
+  qemu:
+    schema: mbr
+    bootloader: u-boot
+    structure:
+      - name: ubuntu-seed
+        role: system-seed
+        filesystem: vfat
+        type: 0C
+        size: 1200M
+        content:
+          - source: boot.scr
+            target: /boot.scr
+      - name: ubuntu-boot
+        role: system-boot
+        filesystem: vfat
+        type: 0C
+        size: 750M
+        content:
+          - source: boot.sel
+            target: uboot/ubuntu/boot.sel
+      - name: ubuntu-save
+        role: system-save
+        filesystem: ext4
+        type: 83,0FC63DAF-8483-4772-8E79-3D69D8477DE4
+        size: 16M
```

Comment:
32M better, we had problems with encryption and arm64 using 4k disk sectors in the past